src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -80,6 +80,10 @@ def validate_lab_vm_list(namespace):
         namespace.filters = "Properties/ownerObjectId eq '{}'".format(object_id)
 
 
+def validate_user_name(namespace):
+    namespace.user_name = "@me"
```

Comment:
If this command is reflecting on the SDK, then this is perfectly acceptable. If it calls into a custom command, then you can just omit that parameter in the signature and plug in "@me" where appropriate.